### DISCUSSION TITLE
Fortescue in andes

### DISCUSSION THREAD

**Discussion by Adarsh-Tayde** [CURENT/andes#507] [Q&A]
can anyone explain how Fortescue work in andes with small example I am new to this thanks

**Comment by jinningwang** [comment]
I don't have much experience with it, but there is an example input file, https://github.com/CURENT/andes/blob/master/andes/cases/5bus_fortescue.xlsx

Welcome further discussion if you have more details or more specific questions.

**Comment by Adarsh-Tayde** [comment]
ok Thanks:)

**Comment by cuihantao** [comment]
It is a symmetric component model. You can check the 1918 Fortescue paper.
